samples/boards/nordic/system_off/src/main.c

```diff
@@ -74,6 +80,10 @@ int main(void)
 
 	printf("Entering system off; press sw0 to restart\n");
 #endif
+#if defined(CONFIG_LPCOMP_WAKEUP_ENABLE)
+	comparator_set_trigger(comp_dev, COMPARATOR_TRIGGER_BOTH_EDGES);
```

Comment:
the comparator could already be pending here, it should be cleared before going to sleep. Call `(void)comparator_trigger_is_pending()` to clear the pending status.

Reply:
done